submissions/xT0PERx/js-dom/index.html

```diff
@@ -11,18 +11,16 @@
       <nav class="menu">
         <h2>О компании :</h2>
         <ul class="menu-list">
-          <li class="menu-item-1">audi</li>
-          <li class="menu-item-2">mersedes</li>
-          <li class="menu-item-3">bmw</li>
-          <li class="menu-item-4">nissan</li>
-          <li class="menu-item-5">tesla</li>
+          <li data-item="menu-item-1" class="item-1">audi</li>
+          <li data-item="menu-item-2" class="item-2">mersedes</li>
+          <li data-item="menu-item-3" class="item-3">bmw</li>
+          <li data-item="menu-item-4" class="item-4">nissan</li>
+          <li data-item="menu-item-5" class="item-5">tesla</li>
```

Comment:
In case when you use data-attribute with a custom identifier, no need to use custom class name for each item.
simple `.item` will be enough.

Reply:
Again, my inattention, it seems like I'm already sitting and looking - I think at least again I shouldn't miss some trifle - and as a result I still miss it. I really hope that with experience I will begin to catch such things.